File: app/src/main/java/com/evaluation/network/RestApi.java
package com.evaluation.network;

import com.evaluation.model.UserList;

import io.reactivex.Single;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface RestApi {

    @GET("api/")
    Single<UserList> getUserList(@Query("results") int results);

}

File: app/src/main/java/com/evaluation/network/RestApi.kt
package com.evaluation.network

import com.evaluation.model.UserList
import io.reactivex.Observable
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Query

interface RestApi {

    @GET("api/")
    fun getUserList(@Query("results") results: Int): Observable<UserList>

}